### DISCUSSION TITLE
Add ability to see the EXIF metadata for attachments

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5871] [Feature Requests]
<img width="1131" alt="image" src="https://github.com/specify/specify7/assets/37256050/37400c9e-f5d0-4597-a1e5-76572bb5812b">

This was done in Specify 6. It should be easy for the user to see the EXIF information embedded in the image itself from the user interface in Specify 7.

**Comment by realVinayak** [comment]
Asset server already has a endpoint for this https://github.com/specify/web-asset-server/blob/ae6c660204652986b7cdfb493de06823356ef04c/server.py#L305
